src/main/java/thePackmaster/patches/MainMenuUIPatch.java

```diff
@@ -84,7 +90,7 @@ public class MainMenuUIPatch {
             dropdowns.add(d);
 
             if (i == 0) {
-                d.setSelectedIndex(2);
+                d.setSelectedIndex(coreSetPackIndex);
```

Comment:
This code must have been written assuming that the core set pack was always the first one in the `SpireAnniversary5Mod.allPacks` list. That was true before automatic pack registration, but isn't any more.

Those changes were being made around the same time (in the prep work ahead of opening the repo up to contributions), so this likely worked locally but was broken as soon as it and automatic pack registration were both on the main branch.